### PR TITLE
[Siyahmelek] add website support (#1625)

### LINKED ISSUE #1625: add siyahmelek
> **From:** 

**Version:** [](https://github.com/manga-download/hakuneko/commits/)

https://siyahmelek.com/
manga site in Turkish
theme: Wordpress Madara

<details><summary>What has been done?</summary>

</details>

### PR DESCRIPTION
**Pull request by dodieboy**
Resolves #1625

### CODE REVIEW

**Comment by XenonR** [comment]
Code looks good, but the logo is too big in size and dimension. It should be roughly 1~16 KB @ 64×64 pixel ([Developer Website Tutorial - Website Scraper Logo](https://github.com/manga-download/hakuneko/wiki/Developer-Website-Tutorial#website-scraper-logo))

**Review by XenonR — CHANGES_REQUESTED** [review]
Code looks good, but the logo is too big in size and dimension. It should be roughly 1~16 KB @ 64×64 pixel ([Developer Website Tutorial - Website Scraper Logo](https://github.com/manga-download/hakuneko/wiki/Developer-Website-Tutorial#website-scraper-logo))

File: src/web/mjs/connectors/Siyahmelek.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class Siyahmelek extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'siyahmelek';
+        super.label = 'Siyahmelek';
+        this.tags = [ 'manga', 'webtoon', 'turkish' ];
+        this.url = 'https://siyahmelek.com/';
```
Comment (27136773) [inline src/web/mjs/connectors/Siyahmelek.mjs]: May remove trailing slash

**Comment by 27136773** [comment]
The icon name should match exactly the ID of the connector (case sensitive)

**Review by XenonR — APPROVED** [review]
Perfect

**Pull request merged**